modules/sor/balancer-sor.integration.test.ts

```diff
@@ -64,303 +64,6 @@ describe('Balancer SOR Integration Tests', () => {
         snapshot = await client.snapshot();
     });
 
-    describe('Weighted Pool Path', () => {
```

Comment:
Removed duplicate tests that are now covered by the new integration test structure